public/app/plugins/panel/gauge/panelcfg.cue

```diff
@@ -29,8 +29,9 @@ composableKinds: PanelCfg: {
 					common.SingleStatBaseOptions
 					showThresholdLabels:  bool | *false
 					showThresholdMarkers: bool | *true
-					minVizWidth:          uint32 | *75
-					minVizHeight:         uint32 | *75
+					sizing:               common.BarGaugeSizing & (*"auto" | _)
```

Comment:
We can rename the `BarGaugeSizing` to just `GaugeSizing` to make it more universal if desired - I don't feel too strongly on need to do this 🤷‍♂️